runtime/platform.c

```diff
@@ -135,17 +135,12 @@ void caml_plat_cond_free(caml_plat_cond* cond)
 
 /* Memory management */
 
-static uintnat round_up(uintnat size, uintnat align) {
-  CAMLassert(Is_power_of_2(align));
-  return (size + align - 1) & ~(align - 1);
-}
-
 intnat caml_plat_pagesize = 0;
 intnat caml_plat_mmap_alignment = 0;
 
 uintnat caml_mem_round_up_pages(uintnat size)
 {
-  return round_up(size, caml_plat_pagesize);
+  return caml_round_up(size, caml_plat_pagesize);
```

Comment:
This file is missing a `misc.h` include, maybe?

Reply:
Indeed. Added include and squashed.